Sources/StreamChatUI/CommonViews/Attachments/AttachmentViews/FileAttachmentView.swift

```diff
@@ -0,0 +1,106 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+/// A view that displays the file attachment.
+open class FileAttachmentView: _View, AppearanceProvider {
+    open var height: CGFloat = 54
+    
+    public struct Content {
+        /// The title of the attachment.
+        var title: String
+        /// The size of the attachment.
+        var size: Int64
+        /// The attachment type.
+        var iconName: String?
+        
+        public init(title: String, size: Int64, iconName: String?) {
+            self.title = title
+            self.size = size
+            self.iconName = iconName
+        }
+    }
+    
+    public var content: Content? {
+        didSet {
+            updateContentIfNeeded()
+        }
+    }
+    
+    public private(set) lazy var fileNameLabel = UILabel()
+        .withoutAutoresizingMaskConstraints
+        .withBidirectionalLanguagesSupport
+        .withAdjustingFontForContentSizeCategory
+
+    public private(set) lazy var fileSizeLabel = UILabel()
+        .withoutAutoresizingMaskConstraints
+        .withBidirectionalLanguagesSupport
+        .withAdjustingFontForContentSizeCategory
+    
+    public private(set) lazy var fileNameAndSizeStack: ContainerStackView = {
+        let stack = ContainerStackView(arrangedSubviews: [fileNameLabel, fileSizeLabel])
+        stack.axis = .vertical
+        stack.alignment = .leading
+        stack.spacing = 3
+        return stack.withoutAutoresizingMaskConstraints
+    }()
+
+    /// The image view that displays the file icon of the attachment.
+    public private(set) lazy var fileIconImageView = UIImageView()
+        .withoutAutoresizingMaskConstraints
+
+    override open func setUpAppearance() {
+        super.setUpAppearance()
+        
+        backgroundColor = appearance.colorPalette.background
+        layer.cornerRadius = 15
```

Comment:
As previous comment about unifying the corner radius stated, this would be nice + it feels strange to me that this is `15` and previews are `16`...

Reply:
fixed, thanks!